tests/page/page-event-request.spec.ts

```diff
@@ -98,6 +98,38 @@ it('should report requests and responses handled by service worker with routing'
   expect(await failedRequest.response()).toBe(null);
 });
 
+it('should not intercept requests handled by service worker with routing', async ({ page, server, isAndroid, isElectron, mode, platform, browserName }) => {
```

Comment:
Maybe we should merge this test into the previous one? It seems like just a few more expect calls are needed.